someip2val/README.md

```diff
@@ -50,12 +50,12 @@ SOME/IP feeder is [COVESA / vsomeip](https://github.com/COVESA/vsomeip/) applica
 1. Install and configure conan (if needed)
     ``` bash
     sudo apt-get install -y python3 python3-pip
-    pip3 install conan
+    pip3 install "conan==1.55"
     ```
     **NOTE:** Sometimes latest conan recipe revisions are broken, but the local build succeeds using cached older revision. If build fails on CI local conan cache could be cleared to reproduce the error. Also latest recipes may require newer conan version.
     ``` bash
     rm -rf ~/.conan/data
-    pip3 install -U conan
+    pip3 install -U "conan==1.55"
```

Comment:
-U is redundant with fixed version here, better add to Note that  conan must be < 2.0, due to issues with aarch64 builds.
Another option would be:
```
pip3 install -U "conan<2.0"
```
But there is a hardcoded recipe in `conanfile.txt` that may need attention after updating conan version

Reply:
Ok, how about this?
```suggestion
    pip3 install "conan==1.*"
```
the reason for the PR is that I got conan 2 installen then the build script failed (I think "conan info")

Reply:
Looks fine